hydra-role-management.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'bootstrap_form'
   gem.add_dependency 'bundler', '>= 1.5'
   gem.add_dependency 'cancancan'
-  gem.add_dependency 'json', '~> 1.8'
+  gem.add_dependency 'json'
```

Comment:
should it be `gem.add_dependency 'json', '>= 1.8'`?

Reply:
Great suggestion! I have updated this PR accordingly.